src/components/MovieDetails/MovieSimilar.tsx

```diff
@@ -77,6 +78,11 @@ const MovieSimilar: React.FC = () => {
 
   return (
     <>
+      <PageTitle
+        title={`${intl.formatMessage(messages.similar)}${
+          movieData && ' - ' + movieData.title
```

Comment:
This will print undefined if `movieData` evaluates to undefined. Although it shouldn't be undefined, we should build the string outside of this return.

Reply:
This applies to other files using the same pattern.

Reply:
How about the following?  Or is it still better to define the string outside of the return?
```
      <PageTitle
        title={`${intl.formatMessage(messages.similar)}${
          movieData ? ' - ' + movieData.title : ''
        }`}
      />
```

Reply:
I would build PageTitle to accept an array OR string. if its an array, it would do something like `title.join(' - ')` and solve this problem :D